### PR TITLE
dts: stm32f7-pinctrl Add definitions for F7 USART/UARTs

### PR DESCRIPTION
**Pull request by therealprof**
Signed-off-by: Daniel Egger <daniel@eggers-club.de>

### CODE REVIEW

File: dts/arm/st/stm32f7-pinctrl.dtsi [hunk 1]
```diff
@@ -5,3 +5,101 @@
...
+			};
+			uart7_pins_a: uart7@0 {
+				rx_tx {
+					rx = <STM32_PIN_PE7 (STM32_PINMUX_ALT_FUNC_8 | STM32_PUPDR_NO_PULL)>;
+					tx = <STM32_PIN_PE8 (STM32_PINMUX_ALT_FUNC_8 | STM32_PUSHPULL_NOPULL)>;
+				};
+			};
+			uart7_pins_b: uart7@1 {
+				rx_tx {
+					rx = <STM32_PIN_PE6 (STM32_PINMUX_ALT_FUNC_8 | STM32_PUPDR_NO_PULL)>;
```
Comment (erwango) [inline dts/arm/st/stm32f7-pinctrl.dtsi]: STM32_PIN_PF6

**Review by erwango — CHANGES_REQUESTED** [review]
One comment otherwise sounds good.
Then please add a one liner comment in the commit message.

**Comment by therealprof** [comment]
@erwango Done, thanks for the review.

**Comment by therealprof** [comment]
@erwango Would you mind re-checking whether everything is to your satisfaction, please?

**Review by erwango — APPROVED** [review]

**Review by ydamigos — APPROVED** [review]

**Pull request merged**